src/middleware/metrics.js

```diff
@@ -3,7 +3,7 @@
 const metrics = require('../lib/metrics');
 const register = metrics.promClient.register;
 
-module.exports = (req, res, next) => {
+module.exports = async (req, res) => {
```

Comment:
Have you been able to test this locally? Because I don't think you need the async here - that will make this function return a promise, which means that you would have to deal with that when it's called later on.

Reply:
You could also add a test to make sure that the return is correct

Reply:
I've been able to test this locally in the sense that when I spun `lev-api` in docker and k8s that it would now return metrics on the `/metrics` endpoint.

I thought that when you set the function as sync that it returned a promised (which is what is was set to before)? But it sounds like I'm wrong about that

I think there could be value in adding that unit test, so I'll do that now

Reply:
so it turns out that previously it was a sync function which was breaking `lev-api`
the changes made here in-fact change the return type to a promise
the unit test i've added checks for that now

if i curl localhost:8080/metrics with this in place it returns a payload of metrics, which is happily fed into prometheus as seen yesterday from my testing